.gitignore

```diff
@@ -0,0 +1,2 @@
+nova-compute-powerflex_ubuntu-22.04-amd64-s390x-ppc64el-arm64.charm
+powerflex-config.yaml
```

Comment:
You probably don't want to check in .pyc files, so you should consider adding `**/*.pyc`. Additionally, change the charm line to simply `*.charm` to make it ignore any and all charm files.